framework/src/utils/JsonSyntaxTree.C

```diff
@@ -174,7 +175,11 @@ JsonSyntaxTree::addParameters(const std::string & parent,
     json["parent_syntax"] = parent;
     json["description"] = params->getClassDescription();
     if (lineinfo.isValid())
+    {
       json["file_info"][lineinfo.file()] = lineinfo.line();
```

Comment:
This it not related to the PR, but doesn't this fail if syntax is defined multiple times in a file?

```
  registerSyntax("MyAction", "Foo/*");
  registerSyntax("MyOtherAction", "Foo/*");
```